packages/trees/frontend/sirius-components-trees/src/treeitems/context-menu/DeleteMenuItem.tsx

```diff
@@ -26,7 +26,7 @@ import {
   GQLErrorPayload,
 } from './DeleteMenuItem.types';
 
-const deleteTreeItemMutation = gql`
+export const deleteTreeItemMutation = gql`
```

Comment:
Please don't share behavior by simply exporting the content of the mutation like that. Create a dedicated hook `useDeleteTreeItem` and reuse it where is is necessary.